tla/ccfraft.tla

```diff
@@ -1202,16 +1202,22 @@ QuorumLogInv ==
             \E j \in S :
                 IsPrefix(Committed(i), log[j])
 
-\* The "up-to-date" check performed by servers
-\* before issuing a vote implies that i receives
-\* a vote from j only if i has all of j's committed
-\* entries
+\* True if server i could receive a vote from server j based on the up-to-date check
+\* The "up-to-date" check performed by servers before issuing a vote implies that i receives
+\* a vote from j only if i has all of j's committed entries
```

Comment:
```suggestion
\* a vote from j only if i has all of j's committable entries
\* See https://github.com/microsoft/CCF/blob/main/src/consensus/aft/raft.h#L1512
```

Reply:
Ok, that makes sense. However a follower wouldn't vote for a candidate that has all of the follower's committed entries, but is missing some of that follower's committable entries in the highest term. So this isn't quite as strong as the condition in the implementation. Not complaining in any way, just checking my understanding :)